doc/portfile.7

```diff
@@ -31,39 +31,49 @@
 .Dd February 13, 2007
 .Dt PORTFILE 7 "MacPorts"
 .Os Darwin
+.
+.
 .Sh NAME
 .Nm Portfile
 .Nd MacPorts description file reference
+.
+.
 .Sh DESCRIPTION
 A complete reference of all available
 .Nm
 variables and example syntax.
 .Nm Portfiles
-consist of valid TCL, allowing for simple key/value pair syntax as well
-as utilization of TCL's extensive functionality.
+consist of valid TCL, allowing for simple key/value pair syntax as well as
+utilization of TCL's extensive functionality.
 .Nm Portfiles
 are encoded in UTF-8.
 .Pp
 The
-\.Nm MacPorts System
-uses a target dependency system based on a
-depends/provides model, allowing for targets to be registered and
-executed in the correct order based on their individual requirements.
+.Nm MacPorts System
+uses a target dependency system based on a depends/provides model, allowing
+for targets to be registered and executed in the correct order based on their
+individual requirements.
```

Comment:
Removing the `\` is probably correct, but what is the purpose of moving the rest from one line to another?

Reply:
Simple formatting to have a right margin of 78 characters, as is common for text documents.  There were many overlong lines that I've also formatted in due course.